.gitkeep

```diff
@@ -1,2 +1,5 @@
 #videos
 routes/videos
+
+#public
+public/
```

Comment:
I already fixed this at https://github.com/gopavasanth/video-cut-tool-back-end/commit/3c452dd142bae7fa4e2d5c8671c1f3f1c8d2dd4f

Reply:
I have merge conflicts that I can't resolve normally.

Reply:
Aah, it's little time taking to resolve such conflicts but I'm sure you will do it sooner :)

Reply:
Yep, finally resolved